### PR TITLE
update vpc flow with flint-s3 based DDL assets and dashboard

### PR DESCRIPTION
**Pull request by YANG-DB**
### Description
Add Flint S3 datasource support for VPC flow logs based integration
- include live query (week's worth data) MV
- include pre-aggregated MV queries
- include pre-aggregated MV window based queries
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_30min_connections_mv-1.0.0.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    status_code,
+    src_endpoint.svc_name,
+    dst_endpoint.svc_name,
+    connection_info['direction']
+  ORDER BY
+    interval_start_time
+WITH (
+    auto_refresh = false
+)
+-- Enable for Auto refresh --
```
Comment (RyanL1997) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_30min_connections_mv-1.0.0.sql:33]: I think we still need to go with the auto refresh here. The empty index issue has been address by using different checkpoints when creating new MV.

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/create_table_parquet_vpc-1.0.0.sql [hunk 2]
```diff
@@ -0,0 +1,75 @@
+ --- DDL VPC create table statement definition
```
Comment (RyanL1997) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/create_table_parquet_vpc-1.0.0.sql:1]: should we include the integration name in the file name? I think for all of the pending PRs, we are just named it like `create_table-1.0.0`

**Review by RyanL1997 — COMMENTED** [review]
Hi @YANG-DB , thanks for putting these together. Just left some minor reviews.

**Comment by Swiddis** [comment]
Also closes https://github.com/opensearch-project/opensearch-catalog/issues/143

**Review by ps48 — APPROVED** [review]

**Review by RyanL1997 — APPROVED** [review]

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_30min_connections_mv-1.0.0.sql [hunk 3]
```diff
@@ -0,0 +1,37 @@
+-- 30 Min Aggregation MV of VPC connections / bytes / packets
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_30min_connections_mv-1.0.0.sql:1]: I think the file gets flattened at some point of install, so comments will cause issues

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_30min_connections_mv-1.0.0.sql [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}_agg_30_min_connections_mview AS
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_30min_connections_mv-1.0.0.sql:2]:
```suggestion
CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}__agg_30_min_connections_mview AS
```

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_60min_connections_mv-1.0.0.sql [hunk 5]
```diff
@@ -0,0 +1,37 @@
+-- One Hour Aggregation MV of VPC connections / bytes / packets
+CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}_agg_60_min_connections_mview AS
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_agg_60min_connections_mv-1.0.0.sql:2]:
```suggestion
CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}__agg_60_min_connections_mview AS
```

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_live_week_mv-1.0.0.sql [hunk 6]
```diff
@@ -0,0 +1,65 @@
+CREATE MATERIALIZED VIEW {table_name}_week_live_mview AS
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_live_week_mv-1.0.0.sql:1]:
```suggestion
CREATE MATERIALIZED VIEW {table_name}__week_live_mview AS
```

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_window-agg_60min_dest_ip_cardinality_mv-1.0.0.sql [hunk 7]
```diff
@@ -0,0 +1,41 @@
+-- One Hour Aggregation time window  of top IP dest by bytes sum group by hourly
+CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}_window_agg_60_min_network_ip_bytes_mview AS
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_window-agg_60min_dest_ip_cardinality_mv-1.0.0.sql:2]:
```suggestion
CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}__window_agg_60_min_network_ip_bytes_mview AS
```

File: server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_window-agg_60min_dest_ip_total-bytes_mv-1.0.0.sql [hunk 8]
```diff
@@ -0,0 +1,41 @@
+-- One Hour Aggregation time window of top IP dest by cardinality group by hourly
+CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}_window_agg_60_min_network_ip_cardinality_mview AS
```
Comment (Swiddis) [inline server/adaptors/integrations/__data__/repository/aws_vpc_flow/assets/vpc_window-agg_60min_dest_ip_total-bytes_mv-1.0.0.sql:2]:
```suggestion
CREATE MATERIALIZED VIEW IF NOT EXISTS {table_name}__window_agg_60_min_network_ip_cardinality_mview AS
```

**Review by Swiddis — APPROVED** [review]

**Review by Swiddis — CHANGES_REQUESTED** [review]
^ Meant to request changes instead of approving

**Review by Swiddis — APPROVED** [review]
Tentatively approving assuming this runs, I haven't tested it locally but I don't see any obvious issues. Seems like active changes are still happening.

**Pull request merged**